batch-job.go

```diff
@@ -30,8 +30,12 @@ import (
 	"time"
 )
 
-// BatchJobType type to describe batch job types
-type BatchJobType string
+type (
+	// BatchJobType describes batch jobs
+	BatchJobType string
+	// BatchJobStatusType describes batch job statuses
+	BatchJobStatusType string
```

Comment:
```suggestion
	BatchJobStatus string
```

Reply:
BatchJobStatus is an existing type; this name would cause a conflict